gladier_tools/posix/tar.py

```diff
@@ -5,12 +5,18 @@ def tar(data):
     import os
     import tarfile
     tar_input = data['tar_input']
+    
     if '~' in tar_input:
         tar_input = os.path.expanduser(tar_input)
+    
+    os.chdir(tar_input)
+    print(os.getcwd())
 
     tar_output = data.get('tar_output', f'{tar_input}.tgz')
-    with tarfile.open(tar_output, "w:gz") as tf:
-        tf.add(tar_input)
+    with tarfile.open(tar_output, 'w:gz') as tf:
+        for file in os.listdir():
+            tf.add(file)
```

Comment:
I think this will error if a user adds a file and not a directory. Can you either make this work for both, or make it raise if a user gives a file for `tar_input`?